ideascube/monitoring/forms.py

```diff
@@ -46,18 +46,19 @@ def clean_serials(self):
         return set([s for s in serials.splitlines() if s])
 
 
-class ExportEntryForm(forms.Form):
+class SinceForm(forms.Form):
 
-    since = forms.DateField(
-        widget=forms.DateInput(format='%Y-%m-%d'),
-        required=False)
+    since = forms.DateField(label=_('since'),
+                            widget=forms.DateInput(format='%Y-%m-%d'),
+                            required=False)
 
 
-class ExportLoanForm(forms.Form):
+class ExportEntryForm(SinceForm):
+    ...
```

Comment:
I'm curious on your reason for using `Ellipsis` here instead of `pass`. :smiley: